### PR TITLE
Update to Python3 and pelican==4.8.0

### PR DESCRIPTION
**Pull request by asmacdo**
- Fixed up some tab/space problems
 - Use unicode to prevent type mismatch
 - Modify pelican conf for 4.8 syntax

Fixup: add __pycache__ to gitignore

### CODE REVIEW

**Comment by asmacdo** [comment]
@yarikoptic

File: pelicanconf.py [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
 
 STATIC_PATHS = [ 'img', 'CNAME' ]
 
 THEME = 'theme'
 
-MD_EXTENSIONS = [ 'codehilite(css_class=highlight)', 'extra', 'mdext.fix_code_blocks' ]
+MARKDOWN = {
+    'extension_configs': {
+        'markdown.extensions.codehilite': {'css_class': 'highlight'},
+        # TODO(asmacdo) I assumed we had to declare fix_code_blocks here, but it doesn't make a change.
```
Comment (asmacdo) [inline pelicanconf.py:21]: @yarikoptic :eyes:

Comment (yarikoptic) [reply pelicanconf.py:21]: Note: that there might be lots of features we don't use. This website/setup was borrowed from the  https://github.com/buddycloud/buddycloud.com (http://buddycloud.com/) and many pages/features disabled, so some settings might not even be pertinent to CON website.

**Review by yarikoptic — APPROVED** [review]
THANK YOU! It opens up opportunities! ;)

**Pull request merged**